ios/RiveReactNativeView.swift

```diff
@@ -181,6 +183,10 @@ class RiveReactNativeView: RCTView, RivePlayerDelegate, RiveStateMachineDelegate
             }
             
             updatedViewModel.layoutScaleFactor = layoutScaleFactor.doubleValue
+            // In React Native we always autobind to true, until we support a more robust data binding API
+            updatedViewModel.riveModel?.enableAutoBind({instance in
+                self.dataBindingViewModelInstance = instance
+            })
```

Comment:
```suggestion
            updatedViewModel.riveModel?.enableAutoBind({ [weak self] instance in
                self?.dataBindingViewModelInstance = instance
            })
```

Reply:
@HayesGordon looks like the introduction of `[weak self]` went into the second usage of this (just below) but not here.